travis/before_install.x86_64-gcc.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -ev
-tar xf sde-external-8.12.0-2017-10-23-lin.tar.bz2
+#tar xf sde-external-8.12.0-2017-10-23-lin.tar.bz2
```

Comment:
Why is this commented out? Is it because the AVX512 tests are run on the jenkins machine? If so, you should remove the line, and possibly also the bits in the travis configuration that use that variable. But in fact, I would like to see this remove of SDE from Travis in a separate PR/patchset, as it is unrelated to adding the purec_scalar targets.

Reply:
I now removed all SDE-related lines